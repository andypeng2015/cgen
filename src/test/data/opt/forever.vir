module forever

export function $foo() {
@start:
  jmp @loop
@loop:
  jmp @loop
}
