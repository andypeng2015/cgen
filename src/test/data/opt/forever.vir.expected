module forever

export function $foo() {
@2:
  jmp @3
@3:
  jmp @3
}
